packages/eslint-plugin-turbo/__tests__/fixtures/workspace/turbo.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
JSONC => JSON

Reply:
Yes, but this isn't JSONC? We support it now, we should test it.

Reply:
Oh hey, look at that. (Goes to scribble over this bit.)